public/app/batch-processor.js

```
@@ -0,0 +1,404 @@
+  function _applyWatermark(data, sampleRate) {
+    // Subtle inaudible watermark: encode a 19kHz tone at -40dB
+    const out = new Float32Array(data.length);
+    const freq = 19000;
+    const amp = 0.01; // -40dB
+    for (let i = 0; i < data.length; i++) {
+      out[i] = data[i] + amp * Math.sin(2 * Math.PI * freq * i / sampleRate);
+    }
+    return out;
+  }
+
+  function _encodeAudio(data, sampleRate, format) {
```

ESLint

[error] 'format' is defined but never used.

(no-unused-vars)